nest-cli.json

```diff
@@ -1,8 +1,19 @@
 {
-  "$schema": "https://json.schemastore.org/nest-cli",
-  "collection": "@nestjs/schematics",
-  "sourceRoot": "src",
-  "compilerOptions": {
-    "deleteOutDir": true
-  }
+    "$schema": "https://json.schemastore.org/nest-cli",
+    "collection": "@nestjs/schematics",
+    "sourceRoot": "src",
+    "compilerOptions": {
+        "deleteOutDir": true,
+        "plugins": [
+            {
+                "name": "@nestjs/swagger",
+                "options": {
+                    "dtoFileNameSuffix": [
+                        "-request.ts",
+                        "-response.ts"
```

Comment:
hmm, I didn't take this into account when dropping the `.dto` 🤔 Let's leave it for now, we will see later if it brings us some trouble or not